test/JDBC/expected/babel_collection.out

```diff
@@ -183,14 +183,15 @@ GO
 ~~START~~
 text
 Query Text: select c1 from testing4 where c1 LIKE 'jones'
-Bitmap Heap Scan on testing4  (cost=11.40..29.53 rows=3 width=32)
-  Filter: ((c1)::text = 'jones'::text)
-  ->  Bitmap Index Scan on c1_idxtesting49a168d73f3ba5aacdfd495b931b8d187  (cost=0.00..11.40 rows=650 width=0)
+Bitmap Heap Scan on testing4  (cost=4.17..11.28 rows=3 width=32)
+  Recheck Cond: (c1 = 'jones'::"varchar")
+  ->  Bitmap Index Scan on c1_idxtesting49a168d73f3ba5aacdfd495b931b8d187  (cost=0.00..4.17 rows=3 width=0)
+        Index Cond: (c1 = 'jones'::"varchar")
```

Comment:
I am not getting this, why isn't it picking simple seq scan or index scan and is picking Bitmap heap scan instead?

Reply:
Can we please check postgres if it shows same behaviour?

Reply:
postgres also shows same behaviour for same input. Based on my observation if the the of rows or the row size is high, the PG usually chooses Index. Else, it may opt for bitmap index scan